app/routes/task_routes.py

```diff
@@ -1 +1,77 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, abort, make_response, Response
```

Comment:
If you look at this import in your code editor, it highlights that `abort` and `make_response` are imported but not accessed so they should be removed to keep our codebase tidy.
```suggestion
from flask import Blueprint, request, Response
```